proto/src/determined/notebook/v1/notebook.proto

```diff
@@ -39,4 +40,8 @@ message Notebook {
   string service_address = 11;
   // The name of the resource pool the Notebook was created in
   string resource_pool = 12;
+  // The config of the notebook;
+  google.protobuf.Struct config = 13;
+  // The exit status;
+  string exit_status = 14;
```

Comment:
question: why not an `int` ? is this because we can't encode lack of a exist code?.

nit: wdy about exit code IMO it's a bit more explicit?

Reply:
update: I see exist_status is defind as a string in our master code. in that case it'd make sense to me to keep it as string especially if we're not sure if it takes non numeric values.